packages/@aws-cdk-testing/framework-integ/test/aws-s3-deployment/test/integ.bucket-deployment-substitution.ts

```diff
@@ -37,11 +37,11 @@ class Test extends cdk.Stack {
 }
 
 const app = new cdk.App();
-new Test(app, 'my-stack');
 
 const testCase = new Test(app, 'test-s3-deploy-substitution');
 const integ = new IntegTest(app, 'deploy-time-substitution-integ-test', {
   testCases: [testCase],
+  diffAssets: true,
```

Comment:
Is this needed? Will hashes change with dependency updates? If so, it we need to disable it.